api/controller/migrationmaster/client.go

```diff
@@ -153,8 +153,8 @@ func (c *Client) ModelInfo() (migration.ModelInfo, error) {
 
 // Prechecks verifies that the source controller and model are healthy
 // and able to participate in a migration.
-func (c *Client) Prechecks() error {
-	return c.caller.FacadeCall("Prechecks", nil, nil)
+func (c *Client) Prechecks(targetControllerVersion version.Number) error {
+	return c.caller.FacadeCall("Prechecks", targetControllerVersion, nil)
```

Comment:
We always pass params as a struct

Reply:
I think we do pass args to API clients. I would prefer to use params if the args get long.

Reply:
Ah, right. I thought you meant `func (c *Client) Prechecks(targetControllerVersion version.Number) error `.
The apiserver side methods should always use params. I missed this piece!